installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -16,76 +16,115 @@ include::modules/oc-mirror-about.adoc[leveloffset=+1]
+If you do not already have an existing solution for a mirror registry, subscribers of {product-title} are provided a xref:../../installing/disconnected_install/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[mirror registry for Red Hat OpenShift]. 
+
+* The mirror registry must be reachable by every machine in the clusters that you provision. If the registry is unreachable, installation, updating, or normal operations such as workload relocation might fail. For this reason, you must run mirror registries in a highly available way, and the mirror registries must at least match the production availability of your OpenShift Container Platform clusters.
+
+* You have set the umask parameter to `0022` on the operating system that uses oc-mirror.
 
 [id="mirroring-preparing-your-hosts"]
 == Preparing your mirror hosts
 
-Before you can use the oc-mirror plugin to mirror images, you must install the plugin and create a container image registry credentials file to allow the mirroring from Red Hat to your mirror.
+In order to use the oc-mirror plugin to mirror images, you must install the plugin and create a container image registry credentials file to allow the mirroring from Red Hat to your mirror.
 
 // Installing the oc-mirror OpenShift CLI plugin
 include::modules/oc-mirror-installing-plugin.adoc[leveloffset=+2]
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../cli_reference/openshift_cli/extending-cli-plugins.adoc#cli-installing-plugins_cli-extend-plugins[Installing and using CLI plugins]
-
 // Configuring credentials that allow images to be mirrored
 include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
 
 // Creating the image set configuration
 include::modules/oc-mirror-creating-image-set-config.adoc[leveloffset=+1]
 
+// Image set configuration parameters
+include::modules/oc-mirror-imageset-config-params.adoc[leveloffset=+2]
+
+// Image set configuration examples
+include::modules/oc-mirror-image-set-config-examples.adoc[leveloffset=+1]
+
+[id="setting-incremental-mirroring"_{context}]
+=== Setting up for incremental mirroring
+
+In most cases, you are mirroring images to install, upgrade, and maintain {product-title} in a disconnected environment. Because the volume of the image set needed to install or upgrade {product-title} can be substantial, oc-mirror keeps metadata about previously mirrored images in its storage backend in order to reduce the volume of the following image sets to the recent images only.
+
+Therefore, you must specify a storage backend in the image set configuration file. The oc-mirror plugin stores metadata in this storage backend during image set creation. You can specify the following storage backends:
+
+* A local directory 
++
+[NOTE]
+====
+This is not recommended for enterprise use cases as it does not scale as well as the registry-based storage backend.
```

Comment:
Can we write something like this then- "This is not recommended for enterprise scenarios because it does not scale as effectively as the registry-based storage backend." or "This is not ideal for enterprise-level companies because it does not scale as effectively as the registry-based storage system."

Reply:
Rephrase this sentence specifying the conditions in which it can't be used. Both the above suggestions don't fit.